hathor/indexes/rocksdb_height_index.py

```diff
@@ -0,0 +1,145 @@
+# Copyright 2021 Hathor Labs
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from typing import TYPE_CHECKING, NamedTuple, Optional, Tuple
+
+from structlog import get_logger
+
+from hathor.conf import HathorSettings
+from hathor.indexes.height_index import HeightIndex
+from hathor.indexes.rocksdb_utils import RocksDBIndexUtils
+from hathor.transaction.genesis import BLOCK_GENESIS
+from hathor.util import not_none
+
+if TYPE_CHECKING:  # pragma: no cover
+    import rocksdb
+
+settings = HathorSettings()
+logger = get_logger()
+
+_CF_NAME_HEIGHT_INDEX = b'height-index'
+
+
+class _IndexEntry(NamedTuple):
```

Comment:
Can we reuse the one from `hathor.indexes.height_index` file here and in the memory height index? If yes, then we should reuse `_IndexEntry` and `BLOCK_GENESIS_ENTRY`

Reply:
Good idea. Done.